server/src/termdb.gdc.js

```diff
@@ -998,8 +999,20 @@ async function cacheSampleIdMapping(ds) {
 	initGdcHolder(ds)
 
 	try {
+		// also indicates to ignore data_release_version even if present
+		const response = await ds.preInit.getStatus()
```

Comment:
when this func runs for first time on server launch, what happens if this line fails?

Reply:
In `mds3.preInit.js` line 238,  `getStatus()` is retried in `mds3.preInit.js` if `ds.preInit.retryMax > 0`, otherwise an error will be thrown immediately after the first try.  Note that for the preInit to not block other ds from loading, the call to `mds3_init.init()` is not awaited on if `ds.loadWithoutBlocking` is true.

Also, in line 1003 of `termdb.gdc.js`, if `getStatus()` fails there, then I think line 1182 will not be called so the automated cache check will stop. I'll think about this when I create `cacheMappingOnNewRelease()`.